aten/src/ATen/cuda/detail/CUDAHooks.cpp

```diff
@@ -459,6 +462,132 @@ void CUDAHooks::deviceSynchronize(DeviceIndex device_index) const {
   c10::cuda::device_synchronize();
 }
 
+void CUDAHooks::getIpcHandleSize(size_t& ipc_memory_handle_size,
+                                 size_t& ipc_event_handle_size) const{
+  // ipc_memory_handle consists of handlel of type cudaIpcMemHandle_t and
```

Comment:
handle

Reply:
@ezyang  CUDAHooks::getIpcHandleSize is deprecated in the latest commit